Лаба 4/task 2.py

```diff
@@ -0,0 +1,20 @@
+import json
+
+
+def task() -> float:
+
+    data = [
+        {"score": 0.0009456152645028281, "weight": 1},
+        {"score": 0.003, "weight": 2},
+        {"score": 0.005, "weight": 3},
+        {"score": 2.274, "weight": 1}
+    ]
+
+    total_sum = 0.0
+    for entry in data:
+        total_sum += entry["score"] * entry["weight"]
+
+    return round(total_sum, 3)
```

Comment:
Тогда решение будет строиться на прочтении файла, получении из него данных и уже их последующая обработка
```suggestion
    with open(filename) as json_file: # открываем файл
        json_data = json.load(json_file) # десериализует строку формата JSON в объект языка Python

    return round(sum([item["score"] * item["weight"] for item in json_data]),3)
```